api/src/opentrons/protocol_api/labware.py

```diff
@@ -280,12 +280,20 @@ def load_liquid(self, liquid: Liquid, volume: float) -> None:
 
         :param Liquid liquid: The liquid to load into the well.
         :param float volume: The volume of liquid to load, in µL.
+
+        .. note::
+            In API Version 2.22 and above, use `load_empty()` to mark a well as empty at the beginning of a protocol.
```

Comment:
```suggestion
            In API version 2.22 and later, use :py:meth:`~.Well.load_empty()` to mark a well as empty at the beginning of a protocol, rather than using this method with ``volume=0``.
```
Yeah, the other method is right there but cross-references are still nice. And I liked the additional info you put in the core docstring, so brought that here as well.

And by precedent we use "and earlier"/"and later" with open ended ranges of API versions.